### PR TITLE
233 feature tests

### PR DESCRIPTION
**Pull request by cdmo**
#233

### CODE REVIEW

**Comment by cdmo** [comment]
Without adding [puffing billy](https://stackoverflow.com/a/17007324/721065), we can't stub JSONP. I am thinking that we just not worry about stubbing for Google Book API's book covers. Probably not worth the complexity of adding another gem and configuring.

SO this PR has no actual mocking after all lol

**Comment by cdmo** [comment]
For this particular spec to work, we'd also need to figure out a way to load Solr fixture data

**Comment by cdmo** [comment]
Ok, updated the fixtures and made CI (Travis) run an index on that, then clear it at the end (not sure that's really necessary though). Think this is ready to have WIP removed.

File: spec/features/book_covers_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.feature 'BookCovers', type: :feature do
+  describe 'User searches for a book' do
+    it 'a book that happens to have a book cover in the Google Books API', js: true do
```
Comment (banukutlu) [inline spec/features/book_covers_spec.rb]:
```suggestion
    it 'that happens to have a book cover in the Google Books API', js: true do
```

Comment (banukutlu) [reply spec/features/book_covers_spec.rb]: minor update ☝️

**Pull request merged**